app/javascript/components/CreateWork.jsx

```diff
@@ -54,20 +54,17 @@ class CreateWork extends React.Component {
         "X_CSRF-Token": document.getElementsByName("csrf-token")[0].content
       }
     }).then((data) => {
-      window.location = `/artists/` + this.props.artist.id;
+
+
+
     }).catch((data) => {
       console.error(data);
     });
   }
 
   onDrop(images) {
-    var currentImages = this.state.work.images;
-    const [newImages] = images;
-    currentImages.push(newImages);
-
-    // Assign files dropped into component into state
     var work = this.state.work;
-    work.images = currentImages;
+    work.images = images
```

Comment:
`ArtistWorks.jsx` errors out because attachments aren't being serialized and because seeds don't have attachments. Can we get rid of these errors?

Reply:
i think we are also good here